integration/terraform/ec2/linux/README.md

```diff
@@ -19,3 +19,64 @@ AMI builder pipeline builds the ami
 The pipeline installs required packages and updates ami software
 
 This process generates a new ami we can then use for testing
+
+**How to integration test in your aws account**
+1. Create resources and setup local
+   1. Install terraform
```

Comment:
Should there be a step for installing localstack?

Reply:
No. Local stack run on an ec2 instance. The instance ami has docker already installed.